examples/qml_extension_plugin/tests/CMakeLists.txt

```diff
@@ -19,12 +19,26 @@ target_link_libraries(${APP_NAME} Qt::QuickTest)
 
 add_dependencies(${APP_NAME} core_qmlplugin)
 
-set(test_cmd "./${APP_NAME}" -import "${CMAKE_CURRENT_BINARY_DIR}/../qml" -input "${CMAKE_CURRENT_SOURCE_DIR}")
-add_test(NAME ${APP_NAME}
+set(test_cmd
+    "./${APP_NAME}"
+    -import
+    "${CMAKE_CURRENT_BINARY_DIR}/../qml"
+    -input
+    "${CMAKE_CURRENT_SOURCE_DIR}"
+    -plugins
+    "$<TARGET_FILE_DIR:core_qmlplugin>"
+)
+
+add_test(
+    NAME ${APP_NAME}
     COMMAND ${test_cmd}
     WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
 )
 
+set_tests_properties(
```

Comment:
```suggestion
# RUNTIME_ENV comes from the CMakeLists.txt at the root of this repository.
set_tests_properties(
```

We've been writing the CMakeLists.txt within the examples so they could stand on their own outside of this repo, but it would be overkill to setup `RUNTIME_ENV` repeatedly just for the tests. So let's leave a comment for people who may be reading this as an example of how to setup their own repository using CXX-Qt.

Reply:
My idea was to later propose to have the cmakelists as non standalone. And create a standalone folder in examples folder with really standalone, not including test and stuff.